scripts/regenerate_docs

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 
-docker pull pseudomuto/protoc-gen-doc
+# docker pull pseudomuto/protoc-gen-doc
```

Comment:
Usually it's better to remove commented code. You can find it in history.

Reply:
This code is still needed as part of the release process, but the exit code below forces me to remember to update to the forked documentation generation beforehand.

Unusual but effective!